File: JavaScript/Fibonacci.js
console.log("Sequência de Fibonacci.");
let n;

do {
    n = parseInt(prompt("Digite quantos termos deseja gerar (N > 1): "));
    if (n <= 1) {
        console.log("N precisa ser maior que 1. Tente novamente.");
    }
} while (n <= 1);

let a = 0, b = 1;
let sequencia = "";

for (let i = 0; i < n; i++) {
    sequencia += a + " ";
    let proximo = a + b;
    a = b;
    b = proximo;
}

console.log("Sequência: " + sequencia + "\n");

File: java/Fibonacci.java
import java.util.Scanner;

public class Fibonacci{
    void gerarFibonacci(){
        Scanner dado = new Scanner(System.in);
        System.out.println("Sequência de Fibonacci.");
        int n;

        do{
            System.out.println("Digite quantos termos deseja gerar (N > 1): ");
            n = dado.nextInt();
            if (n <= 1) {
                System.out.println("N precisa ser maior que 1. Tente novamente.");
            }
        }while (n <= 1);

        int a = 0, b = 1;

        System.out.print("Sequência: ");
        for (int i = 0; i < n; i++) {
            System.out.print(a + " ");
            int proximo = a + b;
            a = b;
            b = proximo;
        }
        System.out.println("\n");
    }
}